tests/fsdp/test_fsdp.py

```diff
@@ -286,6 +286,7 @@ def test_cpu_ram_efficient_loading(self):
         assert os.environ.get("FSDP_CPU_RAM_EFFICIENT_LOADING") == "False"
 
 
+@run_first
```

Comment:
What's the reason why we need this to run first?

Reply:
basically any test that `execute_subprocess_async` or `run_command` will create a subprocess that will try acquiring the hpu device at some point. If the device is already acquired (kinda like if cuda context is initilized) by the current default process (because a previous test allocated some tensor on hpu), the subprocess will fail during device acquisition.